contrib/platform/test/com/sun/jna/platform/win32/IPHlpAPITest.java

```diff
@@ -131,8 +131,8 @@ public void testGetNetworkParams() {
             if (nullPos != -1) {
                 addr = addr.substring(0, nullPos);
             }
-            // addr is now a dotted-notation IP string. Test valid
-            assertTrue(ValidIP.matcher(addr).matches());
+            // addr is now a dotted-notation IP string. Test valid - address can be empty if DNS is IPv6
+            assertTrue("Address not valid: " + addr, ValidIP.matcher(addr).matches() || addr.isEmpty());
```

Comment:
Where's this error from? If it's from the OS, it may be translated on non-English ones.

Reply:
This was an invalid test setup on my side. When you have no network connection, the `DnsServerList` holds an empty entry. My VirtualBox setup was unstable. Will be reverted.